composer.json

```diff
@@ -12,7 +12,7 @@
         "sort-packages": true
     },
     "require": {
-        "php": "7.0.2|7.0.4|~7.0.6|~7.1.0",
+        "php": "~7.1.3||~7.2.0",
```

Comment:
Related https://github.com/magento-engcom/php-7.2-support/pull/30
@pmclain , could you also update the related PR to reflect same PHP versions and target it to the same `upgrade-libs` branch? I'll merge both then. Or if you have time, merge it into this PR.

Reply:
I've applied the updates from #30 to this PR. I'll close #30